interface.cpp

```diff
@@ -264,9 +277,6 @@ void Collection::init_from_json(const Json::Value& config)
         }
         mCollectorMap[c->name()] = c;
     }
-
-    // Various specializations
-    mCollectorMap["battery_temperature"]->doubleTransform(0.1); // divide by 10 and store as float
```

Comment:
it should not be removed.

Reply:
Yeah, I have added this collector when per api doesn't enable in my new commit.